python/src/ExtremeValueCopula_doc.i.in

```diff
@@ -0,0 +1,60 @@
+%feature("docstring") OT::ExtremeValueCopula
+"ExtremeValue copula.
+
+Available constructor:
+    ExtremeValueCopula(*pickandFunction*)
+
+Parameters
+----------
+pickandFunction : :class:`~openturns.Function`
+    Parameter :math:`A`, the Pickand function defining the extreme value copula. By default, it is the constant function equal to 1.
+
+Notes
+-----
+The ExtremeValue copula is a bivariate copula defined by:
+
+.. math::
+
+    C(u_1, u_2) = \exp\left[\log(u_1u_2)A\left(\dfrac{\log u_2}{\log u_1u_2}\right)\right]
+
+for :math:`(u_1, u_2) \in [0, 1]^2`
+
+Where :math:`A` is a convex function satisfying :math:`A(0)=A(1)=1`, :math:`\max(t, 1-t)\leq A(t)\leq 1`.
+
+See also
+--------
+Copula
+
+Examples
+--------
+Create a distribution:
+
+>>> import openturns as ot
+>>> copula = ot.ExtremeValueCopula(ot.SymbolicFunction('t', 't'))
```

Comment:
Identity function does not meet above requirements.

Reply:
Right, I intended to write `ot.SymbolicFunction('t', '1')`, which corresponds to the independent copula. Fixed.